src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -39,6 +40,9 @@ class HaConfigDashboard extends LitElement {
         ${this.hass.localize("ui.panel.config.introduction")}
       </div>
 
+      ${isComponentLoaded(this.hass, "hassio")
+        ? html`<ha-config-updates .hass=${this.hass}></ha-config-updates>`
```

Comment:
Let's add this to the left so it doesn't push the menu down.
```suggestion
        ? html`<ha-config-updates .hass=${this.hass} slot="introduction"></ha-config-updates>`
```